proxy.go

```diff
@@ -696,6 +702,31 @@ func proxyMain() {
 		fmt.Fprintln(os.Stderr, "init:", err.Error())
 		os.Exit(1)
 	}
+
+	// Tune KSM if available
+	if err := proxyKSM.init(defaultKSMRoot); err != nil {
+		fmt.Fprintln(os.Stderr, "init:", err.Error())
```

Comment:
I see a KSM init fail is not fatal (which I think would be correct if the host system does not have KSM available). Is a print to stderr a bit harsh though - should we be using the logger here?
Difficult call I think - hard to differentiate between 'expected as the host has no KSM' or 'was not expecting KSM to fail' I suspect.
I'm just a bit wary that if we have KSM on as the default in the proxy, any user that runs up CC on a system that does not have KSM available will be getting stderr 'errors'.